packages/jsx-email/src/components/conditional.tsx

```diff
@@ -45,13 +57,28 @@ export const Conditional: JsxEmailComponent<ConditionalProps> = (props) => {
       'jsx-email: Conditional expects the `expression` or `mso` prop to be defined, not both'
     );
 
-  return (
-    <>
-      <Suspense fallback={<div>waiting</div>}>
-        <Renderer {...props}>{children}</Renderer>
-      </Suspense>
-    </>
-  );
+  // Build a lightweight marker element that the rehype plugin will transform
+  // into a conditional comment block. We do not inline HTML here.
+  const attrs: ConditionalElementProps = {};
+  if (typeof mso === 'boolean') attrs['data-mso'] = mso ? 'true' : 'false';
+  if (typeof expression === 'string') attrs['data-expression'] = expression;
+
+  // Emit a marker element that the rehype plugin will transform.
+  // - In <head>, use a <meta> marker because custom elements are not
+  //   allowed in <head> in many HTML parsers. This ensures the marker stays
+  //   in the head section through parsing.
+  // - Elsewhere, emit a lightweight custom element.
+  if (head) {
+    return (
+      <meta
+        data-jsx-email-cond="true"
+        {...(attrs['data-mso'] ? { 'data-mso': attrs['data-mso'] } : {})}
+        {...(attrs['data-expression'] ? { 'data-expression': attrs['data-expression'] } : {})}
+      />
+    );
+  }
```

Comment:
this is just awful. revert this mess. we dont care about OTHER parsers. we're only using rehype.